source/fuzz/fuzzer_pass_permute_function_variables.cpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2019 Google LLC
```

Comment:
As you have made this contribution and you don't work for Google, the copyright holder is you. Also, the year is 2021. So this should read:

// Copyright (c) 2021 [Your name]

Reply:
done